schemas.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+
+from pydantic import BaseModel
+from typing_extensions import List
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+    class Config:
+        from_attributes = True
+
+
+class BookCreate(BookBase):
+    author_id: int
+
+
+class BookResponse(BookBase):
+    id: int
+
+    class Config:
+        from_attributes = True
+
+
+class BookListResponse(BaseModel):
+    books: List[BookResponse]
+
+    class Config:
+        from_attributes = True
+
+
+class AuthorBase(BaseModel):
+    id: int
```

Comment:
The `id` field should not be part of the `AuthorBase` class if it's intended to be used for creation purposes. Consider moving the `id` field to the `AuthorResponse` class only, as it represents a database-generated value.